alertwildfire_live_recording.py

```diff
@@ -113,14 +95,16 @@ def capture_and_record(googleServices, dbManager, outputDir, camera_name):
 
 
 
-def fetchAllCameras(googleServices, camera_names_to_watch):
+def fetchAllCameras(camera_names_to_watch):
     """manages the continual observation of a given set of cameras to watch.
     Args:
-        googleServices: google service service (from getGoogleServices())
         camera_names_to_watch (List): list of camera names that are to be watched by this process
     Returns:
         None
     """
+    googleServices = goog_helper.getGoogleServices(settings, [])
```

Comment:
Does moving this code here fix the permission issue you were having with parallel processing?  If so, please add a comment mentioning this, so someone doesn't move it back out in the future.